DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateGroups/GroupCourses.cshtml

```diff
@@ -0,0 +1,73 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateGroups
+@using Microsoft.Extensions.Configuration
+@model GroupCoursesViewModel
+
+@{
+  ViewData["Title"] = "Group courses";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
+  ViewData["HeaderPathName"] = "Tracking System";
+}
+
+<link rel="stylesheet" href="@Url.Content("~/css/trackingSystem/delegates/groupDelegates.css")" asp-append-version="true">
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+@section NavBreadcrumbs {
+  <partial name="_DelegateGroupBreadcrumbs" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-one-quarter">
+    <nav class="side-nav-menu" aria-label="Side navigation bar">
+      <partial name="_DelegateGroupsSideNavMenu" model="Model.NavViewModel" />
+    </nav>
+  </div>
+
+  <div class="nhsuk-grid-column-three-quarters">
+
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-two-thirds">
+        <h1 id="page-heading" class="nhsuk-heading-xl">Group courses</h1>
+      </div>
+      <div class="nhsuk-grid-column-one-third all-delegates-button-group">
+        <a class="nhsuk-button all-delegates-button" role="button">
+          Add
+        </a>
+      </div>
+    </div>
+
+    <form method="get">
+      <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-full">
+          @if (!Model.GroupCourses.Any())
+          {
+            <p class="nhsuk-u-margin-top-4" role="alert">
+              <b>No courses found.</b>
+            </p>
+          }
+          else
+          {
+            <div>

```

Comment:
Yes, to keep the html consistent with other pages, usually it would have id="searchable-elements", but since both this page and the group delegates page don't have search/sort/filter, Steve asked me to remove it from the Group Delegates page, and so here for consistency